src/prerender.ts

```diff
@@ -257,7 +257,10 @@ function extractLinks(
 
   // Extract from x-nitro-prerender headers
   const header = res.headers.get("x-nitro-prerender") || "";
-  _links.push(...header.split(",").map((i) => i.trim()));
+  _links.push(...header.split(",")
+      .map((i) => i.trim())
+      .map((i) => decodeURIComponent(i));
```

Comment:
you are missing a ) here:
```suggestion
      .map((i) => decodeURIComponent(i)));
```

Reply:
Oh my bad. Fixed it in #800. 😔